docs/features.rst

```diff
@@ -109,44 +111,74 @@ description of the available features, see  :ref:`afqmc`.
 -  Specialized implementation for solids with k-point symmetry (e.g.
    primitive unit cells with k-points).
 
-.. _legacygpufeatures:
+
+.. _gpufeatures:
 
 Supported GPU features for real space QMC
 -----------------------------------------
 
-The "legacy" GPU implementation for real space QMC uses NVIDIA CUDA and achieves very good speedup on NVIDIA GPUs. However, only a
-very limited subset of features is available. As detailed in :ref:`roadmap`, a new full-featured GPU implementation is currently
-being made that is both high performing and portable to other GPU architectures. The existing implementation supports multiple
-GPUs per node, with MPI tasks assigned in a round-robin order to the GPUs. Currently, for large runs, 1 MPI task should be used
-per GPU per node. For smaller calculations, use of multiple MPI tasks per GPU might yield improved performance.
-
-Supported GPU features:
-
--  VMC, wavefunction optimization, DMC.
+There are two GPU implementations in the code base.
 
--  Periodic and open boundary conditions. Mixed boundary conditions are
-   not supported.
+  - **Performance portable implementation** (recommended). It implements real space QMC methods
+    using OpenMP offload programming model and accelerated linear algebra libraries.
+    It runs with good performance on NVIDIA and AMD GPUs and the Intel GPU support is under development.
+    Unlike the "legacy" implementation, it is feature complete
+    and users may mix and match CPU-only and GPU-accelerated features.
 
--  Wavefunctions:
+  - **Legacy implementation** fully based on NVIDIA CUDA. It achieves very good speedup on NVIDIA GPUs.
+    However, only a very limited subset of features is available.
 
-   #. Single Slater determinants with 3D B-spline orbitals.
-      Twist-averaged boundary conditions and complex wavefunctions are
-      fully supported. Gaussian type orbitals are not supported.
 
-   #. Hybrid mixed basis representation in which orbitals are
-      represented as 1D splines times spherical harmonics in spherical
-      regions (muffin tins) around atoms and 3D B-splines in the
-      interstitial region.
+The existing implementation supports multiple GPUs per node, with MPI tasks assigned in a round-robin order to the GPUs.
```

Comment:
In my mind, "supporting multiple GPUs per node" implies supporting multiple GPU's per MPI rank.  Supporting one MPI rank for 1 GPU seems like it ought to be the bare minimum - any code that does MPI and uses a GPU should work in that mode.

Reply:
This was an old line. Updated it and added more detail about how the primary GPU is assigned under ` Running on GPU machines `